README.md

```diff
@@ -1,104 +1,113 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+# ibm-management-ingress-operator 
 
-- [Management Ingress Operator](#ibm-management-ingress-operator)
-    - [Overview](#overview)
-    - [Prerequisites](#prerequisites)
-        - [PodSecurityPolicy Requirements](#podsecuritypolicy-requirements)
-        - [PodSecurityPolicy Requirements](#securitycontextconstraints-requirements)
-    - [Getting Started](#getting-started)
-        - [Cloning the repository](#cloning-the-repository)
-        - [Building the operator](#building-the-operator)
-        - [Installing](#installing)
-        - [Uninstalling](#uninstalling)
-        - [Troubleshooting](#troubleshooting)
-        - [Running Tests](#running-tests)
-        - [Development](#development)
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+You can use the ibm-management-ingress-operator to install the IBM System Management ingress service. You can use IBM System Management ingress service to expose your services of the IBM Cloud Paks and IBM Cloud Platform Common Services.
```

Comment:
You can use the ibm-management-ingress-operator to install the IBM System Management ingress service. You can use the IBM System Management ingress service to expose your services when you install and use an IBM Cloud Pak or IBM Cloud Platform Common Services.

Reply:
Updated.